packages/cli/src/cli/utilities/help.ts

```diff
@@ -0,0 +1,33 @@
+import {Command, Help, CommandHelp} from '@oclif/core'
+import chalk from 'chalk'
+
+const {dim} = chalk
+
+class CustomCommandHelp extends CommandHelp {
+  protected override flags(flags: Command.Flag.Any[]): [string, string | undefined][] | undefined {
```

Comment:
how come the typing here is `[string, string | undefined]` instead of just `[string | undefined]`?